examples/tutorial_examples/add_custom_bundle.py

```diff
@@ -0,0 +1,233 @@
+# Custom bundle definitions such as the SLF or OR, and the standard BundleDict can be
+# combined through addition. To get both the SLF and the standard bundles, we
+# would execute the following code::
+#
+#     bundles = bundles + abd.BundleDict()
+#
+# In this case, we will skip this and generate just the SLF.
+
+
+#############################################################################
+# Define GroupAFQ object
+# ----------------------
+# HBN POD2 have been processed with qsiprep [4]_. This means that a brain mask
+# has already been computed for them. As you can see in other examples, these
+# data also have a mapping calculated for them, which can also be incorporated
+# into processing. However, in this case, we will let pyAFQ calculate its own
+# SyN-based mapping so that the `combine_bundle` method can be used below to
+# create a montage visualization.
+#
+# For tractography, we use CSD-based probabilistic tractography seeding
+# extensively (`n_seeds=4` means 81 seeds per voxel!), but only within the ROIs
+# and not throughout the white matter. This is controlled by passing
+# `"seed_mask": RoiImage()` in the `tracking_params` dict. The custom bundles
+# are passed as `bundle_info=bundles`. The call to `my_afq.export_all()`
+# initiates the pipeline.
+
+brain_mask_definition = ImageFile(
+    suffix="mask",
+    filters={'desc': 'brain',
+             'space': 'T1w',
+             'scope': 'qsiprep'})
+
+my_afq = GroupAFQ(
+    bids_path=study_dir,
+    preproc_pipeline="qsiprep",
+    output_dir=op.join(study_dir, "derivatives", "afq_slf"),
+    brain_mask_definition=brain_mask_definition,
+    tracking_params={"n_seeds": 4,
+                     "directions": "prob",
+                     "odf_model": "CSD",
+                     "seed_mask": RoiImage()},
+    clean_params={"clean_rounds": 20,
+                  "length_threshold": 3,
+                  "distance_threshold" : 3,
+                  },
+    segmentation_params = {"parallel_segmentation": {"engine":"serial"}},
+    bundle_info=bundles)
+
+# Redo everying related to bundle recognition. This is useful when changing the bundles.
+# The options for dependent_on are 'track' (to start over from tractography) or 'recog'
+# to start over from bundle recognition
+# my_afq.clobber(dependent_on='recog')
+
+my_afq.export_all()
+
+#############################################################################
+# Visualize a montage
+# ----------------------
+# One way to examine the output of the pyAFQ pipeline is by creating a montage
+# of images of a particular bundle across a group of participants. In the montage function
+# the first input refers to a key in the bundlediect and the second gives the layout
+# of the figure (eg. 2 rows 4 coluns) and finally is the view.
+#
+# .. note::
+#
+#   The montage file is copied to the present working directory so that it gets
+#   properly rendered into the web-page containing this example. It is not
```

Comment:
Are we actually planning to run this example when building the docs? The changes in "conf.py" suggest that maybe not? If we are not running this in building the documentation, maybe we need to change this a bit.

Reply:
@36000  gave me permission to add wget to dependencies